### PR TITLE
Reply to review

### PR DESCRIPTION
**Pull request by BalestraPatrick**
Only missing thing is the styling.

### CODE REVIEW

File: models/Review.js [hunk 1]
```diff
@@ -43,6 +45,15 @@ Review.pre('save', function(next) {
 		}
 	}
 
+	if (this.reply !== undefined) {
+
+		let length = this.reply.text.length;
+
+		if (length > 1000) {
+			this.reply = this.reply.text.substring(0, 999);
```
Comment (larabr) [inline models/Review.js:53]: siete proprio degli admin

**Review by rcotti-jc — COMMENTED** [review]
Database stuff (availability, Day) is out of date, fixed with our PR from DB

**Review by larabr — APPROVED** [review]

File: models/Review.js (hunk 1 above)
Comment (aronfiechter) [reply models/Review.js:53]: Hmm, not cool for users I think, there should be a margin, or something like the Twitter counter on the client side.

Comment (larabr) [reply models/Review.js:53]: we can make it more user friendly later on in case, i mean 1000 chars is a lot

File: models/Review.js [hunk 2]
```diff
@@ -22,6 +23,7 @@ const Review = exports.Review = new mongoose.Schema({
 		text 	: { type: String },
 		score 	: { type: Number, required: true },
 		date	: { type: Date, default: Date.now },
+		reply 	: { type: ObjectID, ref: 'Review' },
```
Comment (aronfiechter) [inline models/Review.js:26]: Is this safe?

File: test/routes-test/freelance-post.js [hunk 3]
```diff
@@ -118,6 +118,45 @@ describe('Freelance-post test: ', function() {
...
+      });
+    });
+
+    it('should respond with a 404 if the given freelance ID is not in the database', function(done) {
+      request(app)
+        .post('/freelance/' + ObjectId().toString() + '/review')
+        .set('Accept', 'application/json')
+        .expect(404, done);
+    });
+
```
Comment (aronfiechter) [inline test/routes-test/freelance-post.js:157]: Could add test (and therefore check in the router) that only `owner` can actually post a reply (by checking `session.username` and `owner` of `freelance`).

**Comment by BalestraPatrick** [comment]
GitHub almost got me, this PR points to master. Let me do one that points to develop.

**Pull request closed without merging**